### ISSUE TITLE
Publish to Pypi

### ISSUE THREAD

**Issue by afoerster** [Cargill/pipewrench#3]

**Comment by morman** [comment]
Might need to adjust the package name for PyPI/pip/whatever as someone is already publishing under "Pipewrench" here:

https://pypi.python.org/pypi/Pipewrench/0.2.1

**Comment by afoerster** [comment]
Yeah I think we'll have to get by with 'pipe-wrench' or 'pipe_wrench'
